dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/io/FileSystemByteSink.java

```diff
@@ -39,6 +39,10 @@ public FileSystemByteSink(@Nonnull Path path) {
 
     @Override
     public OutputStream openStream() throws IOException {
+        Path parent = path.getParent();
+        if (!Files.isDirectory(parent)) {
+            Files.createDirectory(parent);
```

Comment:
Should this be `createDirectories` instead, to create more than one level of parent directory if needed?

Reply:
Right now, I'm more worried about this than having an active test. A unit test on FileSystemByteSink would totally work, though.

Reply:
Code was changed (and we now use `Files.createDirectories` instead :-)